pkgs/applications/science/math/cplex/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv, coreutils, requireFile, patchelf, makeWrapper, openjdk, gtk2, xorg, glibcLocales }:
+
+stdenv.mkDerivation rec {
+  name = "cplex-${version}";
+  version = "128";
+  installer = "cplex_studio${version}.linux-x86-64.bin";
+  
+  src = requireFile rec {
+    name = "${installer}";
```

Comment:
You can just assign `name = "cplex_studio${version}.linux-x86-64.bin";` directly here. And there's no need to use the `$installer` variable down below I'd think